book.toml

```diff
@@ -15,7 +15,7 @@ no-section-label=true
 git-repository-url="https://github.com/rust-lang/rust-forge"
 
 [output.html.redirect]
-"beta-backporting.html" = "/release/beta-backporting.html"
```

Comment:
Shouldn't this one remain and also point to `/release/backporting.html`?

Reply:
hm I have renamed the file from `beta-backporting.md` to `backporting.md` so my understanding is that the new link is `./release/backporting.html` and people having old link pointing to `./release/beta-backporting.html` will be redirected to the new one.
(at least on local it worked like that)

Reply:
There should be two entries, one to handle the URL for the old rename (https://forge.rust-lang.org/beta-backporting.html) and one for the new rename (https://forge.rust-lang.org/release/beta-backporting.html).

```toml
"beta-backporting.html" = "/release/backporting.html"
"release/beta-backporting.html" = "/release/backporting.html"
```

Reply:
@XAMPPRocky pushed the suggested change